pkg/cli/term/reader_windows.go

```diff
@@ -106,11 +106,11 @@ var keyCodeToRune = map[uint16]rune{
 // A subset of constants listed in
 // https://docs.microsoft.com/en-us/windows/console/key-event-record-str
 const (
-	leftAlt   = 0x02
-	leftCtrl  = 0x08
-	rightAlt  = 0x01
-	rightCtrl = 0x04
-	shift     = 0x10
+	leftAlt   uint32 = 0x02
+	leftCtrl  uint32 = 0x08
+	rightAlt  uint32 = 0x01
+	rightCtrl uint32 = 0x04
+	shift     uint32 = 0x10
```

Comment:
Is it necessary to add types to these constants? If not, revert.

Reply:
At some point I had errors that complained about the missing type annotations, but they're not necessary (anymore). I'll remove them